umrr_ros2_driver/CMakeLists.txt

```diff
@@ -70,6 +73,9 @@ rclcpp_components_register_node(smartmicro_radar_node
     EXECUTABLE smartmicro_radar_node_exe
 )
 
+ament_target_dependencies(smartmicro_radar_node
+rclcpp umrr_ros2_msgs)
```

Comment:
Using `ament_auto_find_build_dependencies()` combined with `ament_auto_add_library()` above means this is not necessary.